src/types/Venue.ts

```diff
@@ -95,6 +95,7 @@ export interface Venue {
   termsAndConditions: TermOfService[];
   showRadio?: boolean;
   showBadges?: boolean;
+  accessMode?: string;
```

Comment:
Should this be a specific set of strings? Rather than any string?

Reply:
Yes, but this is pending on #1016